rockcraft.yaml

```diff
@@ -0,0 +1,108 @@
+name: spark
+summary: Spark ROCK
+description: Spark ROCK
+license: Apache-2.0
+
+version: "0.1"
+base: ubuntu:22.04
+platforms:
+  amd64:
+
+entrypoint: ["bash", "/opt/entrypoint.sh"]
```

Comment:
It may we worth including a Pebble based entrypoint. At this point the `pebble` part is automatically included in the rock, and it'll unlock some extra capability for managing the entrypoint when the container is running.

Reply:
Definitely! I have created a item in our spark backlog. Thanks @jnsgruk

Reply:
Thanks Jon and Enrico. Will follow up.